src/tribol/mesh/CouplingScheme.cpp

```diff
@@ -127,6 +127,11 @@ void CouplingSchemeErrors::printMethodErrors()
       SLIC_WARNING_ROOT( "User must call tribol::registerNodalResponse() for each mesh to use this ContactMethod." );
       break;
     }
+    case NULL_REFERENCE_COORDS: {
+      SLIC_WARNING_ROOT(
+          "User must call tribol::registerNodalReferenceCoords() for mesh 2 to use this ContactMethod." );
```

Comment:
Minor: but this warning should apply to friction or tied as well and i'm not sure we require specifically registering reference coords for mesh 2. Tied will be both meshes I believe. Perhaps this warning says: "User must call tribol::registerNodalReferenceCoords() for one or both meshes per registered ContactMethod." At some point documentation will describe what needs to be registered. Thoughts?